RaveAddIn/BaseMaps.xml

```diff
@@ -1,6 +1,28 @@
 ﻿<?xml version="1.0" encoding="utf-8" ?>
 <BaseMaps>
   <Region name="United States">
+	<GroupLayer name="National Wetland Inventory">
+      <Layer name="Wetlands Data" url="https://www.fws.gov/wetlands/arcgis/services/Wetlands/MapServer/WMSServer?">
+        <Metadata>
+          <Meta name="Description">Wetlands Data</Meta>
+        </Metadata>
+      </Layer>
+      <Layer name="Wetlands Raster" url="https://www.fws.gov/wetlands/arcgis/services/Wetlands_Raster/ImageServer/WMSServer?">
+        <Metadata>
+          <Meta name="Description">Wetlands Raster</Meta>
+        </Metadata>
+      </Layer>
+	  <Layer name="Wetlands Status" url="https://www.fws.gov/wetlands/arcgis/services/Wetlands_Status/MapServer/WMSServer?">
+        <Metadata>
+          <Meta name="Description">Wetlands Status</Meta>
+        </Metadata>
+      </Layer>
+      <Layer name="Riparian Data" url="https://www.fws.gov/wetlands/arcgis/services/Riparian/MapServer/WMSServer?">
+        <Metadata>
+          <Meta name="Description">Riparian Data</Meta>
+        </Metadata>
+      </Layer>
+    </GroupLayer>
```

Comment:
The addition of group is fine... if it worked. However, it should not be at top of this list in US. Bury it towards or at bottom on its own. Thanks.